### DISCUSSION TITLE
Git storage pull does not work

### DISCUSSION THREAD

**Discussion by turekg** [requarks/wiki#3711] [Error / Bug Report]
Describe the bug
After updating to  release 2.0.12 git sync'ing has partially stopped working. New pages are showing up in the local git repository but they are not being "digested" . "Force pulling" via the "Import Everything" option on the Storage admin page  results in an the following error in the my apache server log:

[Thu Jan 16 11:45:13.983676 2020] [proxy_http:error] [pid 13534] (20014)Internal error (specific information not available): [client 10.1.235.106:53017] AH01102: error reading status line from remote server 127.0.0.1:3001, referer: https://dais-wiki.mpi-cbg.de/a/storage

Also note that:
- The Git synchronisation status remains green on the Storage admin page, so there is no visual cue that anything is wrong. 
- The "Add Untracked Changes" and "Force Sync" functionalities works, so the it could be that the "Import Everything" failure is unrelated to the problem at hand.

Expected behavior
Expect for new files in repo to show up in the wiki

Host Info (please complete the following information):

    Ubuntu 19.0
    Wiki.js version 2.0.12
    postgres 10.1

I've looked everywhere for logs, but could not find any. Are there any? Is there anything else I can check to find out where the problem lies? Thanks!

**Comment by NGPixel** [comment]
The logs are the output of the node process. So if you defined a service to run node, you need to set the logging options

**Comment by turekg** [comment]
Thanx for that, it turns out I can view the logs with journalctl (systemd). There is a warning in the logs for all the files not being ingested. The warning is the same for all and is like the following:

```
2020-01-15T21:04:31.620Z [MASTER] warn: incomplete explicit mapping pair; a key node is missed; or followed by a non-tabulated empty line at line 1, column 20:
Jan 15 11:02:39 dais-srv1.mpi-cbg.de node[20867]:     title: Activity-log: Transmodal Registration
```

Any ideas?

**Comment by turekg** [comment]
OK.  I can confirm that the problem is that second colon in the line with the title. Someone else is auto generating similar files and has only one colon in all his files. And they are being ingested.

**Comment by mielipuolinen** [comment]
I might have run into this issue too. I've one file that I couldn't pull from git, however wikijs is pushing it just fine. Decided to just copy it manually. Unfortunately I don't have logs to provide.

File/path was named "/CSGO", title "Counter-Strike: Global Offensive", tag "CS:GO". Used markdown formatting, nothing special in the content (texts, links and two links-lists).

Using up-to-date and unmodified docker compose file.

**Comment by turekg** [comment]
@mielipuolinen I am thinking that the problem is like mine, too many colons in the metadata fields confuse the ingester
